libkodimote/libkodimote.pro

```diff
@@ -133,3 +133,8 @@ HEADERS += libkodimote_global.h \
            addonsource.h \
            profiles.h \
            profileitem.h
+
+maemo5 {
+    target.path = /opt/libkodimote/lib
+    INSTALLS += target
+}
```

Comment:
Hmm - won't hurt  to keep it `staticlib` then - as that is describing what it actually is?